docs/alternator/new-apis.md

```diff
@@ -181,27 +181,30 @@ entire data center, or other data centers, in that case.
 
 ## Tablets
 "Tablets" are ScyllaDB's new approach to replicating data across a cluster.
-It replaces the older approach which was named "vnodes". Compared to vnodes,
-tablets are smaller pieces of tables that are easier to move between nodes,
-and allow for faster growing or shrinking of the cluster when needed.
+It replaces the older approach which was named "vnodes". See
+[Data Distribution with Tablets](../architecture/tablets.rst) for details.
 
-In this version, tablet support is incomplete and not all of the features
-which Alternator needs are supported with tablets. So currently, new
-Alternator tables default to using vnodes - not tablets.
+In this version, tablet support is almost complete, so new
+Alternator tables default to following what the global configuration flag
+[`tablets_mode_for_new_keyspaces`](../reference/configuration-parameters.rst#confval-tablets_mode_for_new_keyspaces)
```

Comment:
Consider removing the backquotes for the option name to render as a link.
See https://github.com/scylladb/scylladb/pull/26211/files#r2487197039.

Reply:
Thanks, I'll remove those backquotes in the next version I push.